src/model/series.ts

```diff
@@ -307,6 +341,52 @@ export class Series<T extends SeriesType> extends PriceDataSource implements IDe
 		return this._data;
 	}
 
+	public setCustomConflationReducer<TCustomData>(reducer: (items: readonly TCustomData[]) => TCustomData): void {
+		this._customConflationReducer = reducer as unknown as (items: readonly unknown[]) => unknown;
```

Comment:
```suggestion
	public setCustomConflationReducer(reducer: CustomConflationReducer<unknown>): void {
		this._customConflationReducer = reducer;
```
Refactored so we don't need type assertions